### PR TITLE
update version to beta2

### PR DESCRIPTION
**Pull request by sehz**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Release process
+
+bump up version `VERSION` file
+
+## Release crates
+
+Find crates that has been changed last release
+```
+cargo ws changed
```
Comment (simlay) [inline RELEASE.md:9]: Is this short for https://crates.io/crates/cargo-workspaces#changed?

Comment (sehz) [reply RELEASE.md:9]: yup

Comment (sehz) [reply RELEASE.md]: it would be handy if there are other tools that can show diff between published and what is in the workspace

**Review by simlay — APPROVED** [review]
:+1:

**Pull request merged**